### PR TITLE
Split code

### PR DESCRIPTION
**Pull request by Minglee01**
@incuna/js 

this is a clone of https://github.com/incuna/angular-token-auth/pull/30

but merging into a feature branch

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,48 +1,52 @@
...
+# Upcoming
+
+* BREAKING CHANGE: Seperate the code from the angular_token_auth into seperate files.
+
+## 5.9.0
 
 * Upgrade Angular compatibility to include 1.3, 1.4, and 1.5
 
-# 5.8.0
+## 5.8.0
```
Comment (grahamgilchrist) [inline CHANGELOG.md]: Since this is now a PR into a feature branch, it might have been better to keep these changelog number updates in another PR, but don't worry about it now.

**Comment by grahamgilchrist** [comment]
All looks ok from a quick look. It will help a lot when we get linting and tests in this repo.

**Comment by grahamgilchrist** [comment]
@Minglee01 Is this ok to merge into feature branch or are you still woking on it?

**Comment by Minglee01** [comment]
it doesnt work still.

**Comment by grahamgilchrist** [comment]
What doesn't work? 

Do you mean the `loginForm` directive issue we saw earlier? For some reason the `loginForm` directive no longer appears to be registered. and the directive function is not called.

@Minglee01 Maybe you could test this on another simple site using just this module to check?

**Comment by grahamgilchrist** [comment]
Nice one @Minglee01 👍
